$NetBSD: patch-aj,v 1.3 2012/03/19 12:34:24 joerg Exp $

Force Rpath usage on Linux. For reasons analysed in details,
the binutils detection seems to fail.

--- lnxLib.rules.orig	2004-04-23 20:41:58.000000000 +0200
+++ lnxLib.rules
@@ -12,7 +12,7 @@ XCOMM $XFree86: xc/config/cf/lnxLib.rule
 #endif
 
 #ifndef UseRpath
-#define UseRpath NO
+#define UseRpath YES
 #endif
 
 #ifndef SharedOldX
@@ -69,7 +69,11 @@ XCOMM $XFree86: xc/config/cf/lnxLib.rule
 # endif
 #ifndef RpathLoadFlags
 #if UseRpath
-#define RpathLoadFlags -Wl,-rpath,$(USRLIBDIRPATH)
+#  ifdef RealProjectRoot
+#  define RpathLoadFlags -Wl,-rpath,$(USRLIBDIRPATH) -Wl,-rpath,$(REALPROJECTROOT)/lib -L $(REALPROJECTROOT)/lib
+#  else
+#  define RpathLoadFlags -Wl,-rpath,$(USRLIBDIRPATH)
+#  endif
 #else
 #define RpathLoadFlags /**/
 #endif
@@ -78,7 +82,7 @@ XCOMM $XFree86: xc/config/cf/lnxLib.rule
 #define LibraryRpathLoadFlags RpathLoadFlags
 #endif
 # ifndef SharedLibraryLoadFlags
-#  define SharedLibraryLoadFlags -shared
+#  define SharedLibraryLoadFlags -shared RpathLoadFlags
 # endif
 # ifndef PositionIndependentCFlags
 #  define PositionIndependentCFlags -fPIC
@@ -87,17 +91,7 @@ XCOMM $XFree86: xc/config/cf/lnxLib.rule
 #  define PositionIndependentCplusplusFlags -fPIC
 # endif
 # ifndef ExtraLoadFlags
-#  if LinuxBinUtilsMajorVersion >= 26
-#   ifdef UseInstalled
-#    if LinuxBinUtilsMajorVersion < 27
-#     define ExtraLoadFlags RpathLoadFlags -Wl,-rpath-link,$(USRLIBDIRPATH)
-#    endif
-#   else
 #    define ExtraLoadFlags RpathLoadFlags -Wl,-rpath-link,$(BUILDLIBDIR)
-#   endif
-#  else
-#   define ExtraLoadFlags RpathLoadFlags
-#  endif
 # endif
 # ifndef HardCodeLibdirFlag
 #  define HardCodeLibdirFlag RpathLoadFlags
